tsdfileapi/utils.py

```diff
@@ -99,7 +101,7 @@ def sns_dir(base_pattern, tenant, uri, tenant_string_pattern, test=False):
             return _path
         if not os.path.lexists(_path):
             logging.info('Creating %s', _path)
-            os.makedirs(_path, mode=0o770)
+            os.makedirs(_path)
```

Comment:
Should a line with an `os.chmod()` call be added here or is the umask now set somewhere so that mode setting is not needed?

Reply:
I though we concluded that the mode was ignored in prod?

Reply:
I guess calling `os.chmod` after is a good approach anyways.